web/src/components/ReciterCard.vue

```diff
@@ -3,16 +3,14 @@
     <v-card :class="classObject" :style="{ 'background-color': background }">
       <div class="reciter-card__avatar">
         <v-avatar size="48px">
-          <img crossorigin ref="avatarElement" :src="avatar" :alt="name">
+          <img crossorigin ref="avatarElement" :src="image" :alt="name">
```

Comment:
shouldnt "image" here be "avatar"? I think in database we are calling it avatar?